symengine/ntheory.cpp

```diff
@@ -1474,4 +1474,29 @@ void powermod_list(std::vector<RCP<const Integer>> &pows, const RCP<const Intege
     }
 }
 
+int mobius(const Integer &a)
```

Comment:
Return type as a `Integer` instead of machine int. Would be better in actual usage.